va/meson.build

```diff
@@ -269,6 +282,52 @@ if WITH_WAYLAND
 endif
 
 fs = import('fs')
+if WITH_WIN32
+  libva_win32_sources = [
+    'win32/va_win32.c',
+  ]
+
+  libva_win32_headers = [
+    'win32/va_win32.h',
+  ]
+
+  libva_headers_subproject += libva_win32_headers
+
+  deps = [ libwin32_dep, libva_dep]
+
+  libva_win32_args = [
+    '-DLIBVA_MAJOR_VERSION=@0@'.format(libva_major_version)
+  ]
+
+  install_headers(libva_win32_headers, subdir : 'va')
+
+  # Win32 backend statically linked
+  # Make the main va target a static_library on Windows and have the win32 target generate the final DLL
+  libva_win32 = shared_library(
+    'va',
+    sources : libva_win32_sources +
+              libva_win32_headers,
+    soversion : libva_lt_current,
+    version : libva_lt_version,
+    install : true,
+    c_args : libva_win32_args,
+    vs_module_defs : 'libva.def',
```

Comment:
Building on the above (shared library) comment: libva should expose the core functionality, while libva-platform the platform specific one. As such the libva.def list and handling would need to be tweaked.